test/it/EditVatLinkAcceptanceSpec.scala

```diff
@@ -36,6 +36,36 @@ class EditVatLinkAcceptanceSpec extends ServerAcceptanceSpec with WireMockSbrCon
        |  "audit": { "username": "abcd" }
        |}""".stripMargin
 
+  private val InvalidVATEditParentLinkPostBody =
```

Comment:
Could this not be ```VATEditParentLinkPostBody + "}"```?

That might also make it more obvious what the issue is - rather than the reader having to scan the whole body looking for the problem.